.pre-commit-config.yaml

```diff
@@ -52,7 +52,7 @@ repos:
   - id: build-docs
     language: python
     files: (\.(rst)$)|(^doc)
-    additional_dependencies: [PyYAML==6.0.1, sphinx-ncs-theme==0.6.2]
+    additional_dependencies: [PyYAML==5.4.1, sphinx==4.3.0, sphinx-ncs-theme==0.6.5, sphinx-rtd-theme==0.5.2]
```

Comment:
Is there any reason you downgraded PyYAML?
This version for newer python return errors and pre-commit fails:
![image](https://github.com/NordicSemiconductor/npmx-zephyr/assets/124269170/7aec839e-89fd-4491-b264-729965d734a3)

Reply:
I took those values from this file:
https://github.com/NordicSemiconductor/npmx-zephyr/blob/main/doc/requirements-doc.txt

Reply:
My bad - I took them from main branch instead of develop. Fixed.